recipes/mosdepth/build.sh

```diff
@@ -10,9 +10,10 @@ if [[ ${target_platform}  == osx-64 ]] ; then
     tar -xzf mosdepth-latest.tar.gz
     cd mosdepth-${PKG_VERSION}
```

Comment:
```suggestion
    cd mosdepth-${PKG_VERSION}
    export CFLAGS="$CFLAGS -I$PREFIX/include"
    export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
```
https://bioconda.github.io/contributor/troubleshooting.html#zlib-errors